avocado/core/loader.py

```diff
@@ -122,6 +122,8 @@ def __init__(self):
         self._initialized_plugins = []
         self.registered_plugins = []
         self.reference_plugin_mapping = {}
+        self._label_mapping = None
+        self._decorator_mapping = None
```

Comment:
To store the overall mapping to avoid the need to re-create it with each `get_type_label_mapping/get_decorator_mapping` call.